arroyo/processing/processor.py

```diff
@@ -17,13 +17,48 @@
 
 logger = logging.getLogger(__name__)
 
-LOG_THRESHOLD_TIME = 10  # In seconds
+LOG_THRESHOLD_TIME = 5  # In seconds
 
 
 class InvalidStateError(RuntimeError):
     pass
 
 
+class MetricsBuffer:
+    def __init__(self) -> None:
+        self.__metrics = get_metrics()
+        self.__metrics_frequency_sec = 1.0
```

Comment:
Was this intended to be hardcoded? It is used in one place only. If it was meant to be hardcoded probably you should have a constant. If it was meant to be configurable it should be made configurable.

Reply:
Yes it was meant to be hardcoded. It's only being used here right now.